utils/shared.js

```diff
@@ -1126,6 +1126,35 @@ const sortBoxOnBagRemoval = (boxData, bagsToRemove, currDate) => {
     };
 }
 
+/**
+ * We've had an intermittnt issue with Streck tube data not being added to the specimen data structure (11/2023). 
+ * This is a safety net to ensure the data is added. Build the object literal since we only need to do this for streck tubes.
+ * @param {object} streckTubeData - the streck tube data object (empty placeholder passed in for assignment by reference).
```

Comment:
NIT: it looks like the docblock is missing 1 argument

Reply:
nice catch, fixed